one-ui/src/main/java/com/marklogic/hub/oneui/managers/SavedQueriesManager.java

```diff
@@ -0,0 +1,43 @@
+package com.marklogic.hub.oneui.managers;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.MarkLogicServerException;
+import com.marklogic.hub.HubConfig;
+import com.marklogic.hub.dataservices.SavedQueriesService;
+import com.marklogic.hub.oneui.exceptions.BadRequestException;
+import com.marklogic.hub.oneui.exceptions.DataHubException;
+
+public class SavedQueriesManager {
+
+    private DatabaseClient finalDataServiceClient;
+
+    public SavedQueriesManager(HubConfig hubConfig) {
+        finalDataServiceClient = hubConfig.newFinalClient();
```

Comment:
Does this really work? I'm thinking this will break because newFinalClient() passes in a finalDbName, which will cause Data Services to throw an error. I think you want newFinalClient(null).

Reply:
Also, instead of hanging onto a reference to the DatabaseClient, why not instantiate the SavedQueriesService here?